### PR TITLE
Added unit test for K datetime format

### PR DESCRIPTION
**Pull request by 304NotModified**

### CODE REVIEW

File: tests/NLog.UnitTests/LayoutRenderers/DateTests.cs [hunk 1]
```diff
@@ -58,6 +58,36 @@ public void DefaultDateTest()
...
+        public void TimeZoneTest()
+        {
+            var dateLayoutRenderer = new DateLayoutRenderer();
+          
+            dateLayoutRenderer.Format = "yyyy-MM-ddTHH:mmK";
+
+            var logEvent = new LogEventInfo(LogLevel.Info, "logger", "msg");
+            var result = dateLayoutRenderer.Render(logEvent);
+
+            var offset = TimeZone.CurrentTimeZone;
```
Comment (bhaeussermann) [inline tests/NLog.UnitTests/LayoutRenderers/DateTests.cs]: `offset` is not used.

Comment (304NotModified) [inline tests/NLog.UnitTests/LayoutRenderers/DateTests.cs]: One line below? ;)

Comment (bhaeussermann) [inline tests/NLog.UnitTests/LayoutRenderers/DateTests.cs]: Oh, now I see it :P (although a better variable name may have been used).

**Comment by 304NotModified** [comment]
Agree!

**Pull request merged**